src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Planta.kt

```diff
@@ -1,39 +1,52 @@
 package ar.edu.unahur.obj2.semillasAlViento
 
-abstract class Planta(val anioObtencionSemilla: Int, var altura: Float) {
+abstract class Planta(val anioObtencionSemilla: Int, val altura: Float)
+{
   fun esFuerte() = this.horasDeSolQueTolera() > 10
 
-  fun parcelaTieneComplicaciones(parcela: Parcela) =
-    parcela.plantas.any { it.horasDeSolQueTolera() < parcela.horasSolPorDia }
-
   abstract fun horasDeSolQueTolera(): Int
-  abstract fun daSemillas(): Boolean
+  open fun daSemillas(): Boolean {
+   return this.esFuerte()
+  }
 }
 
 class Menta(anioObtencionSemilla: Int, altura: Float) : Planta(anioObtencionSemilla, altura) {
   override fun horasDeSolQueTolera() = 6
-  override fun daSemillas() = this.esFuerte() || altura > 0.4
+  override fun daSemillas() = super.esFuerte() || altura > 0.4
```

Comment:
Esto está mal, nunca deberías llamar a otro método cuando usas `super`. Lo que sí podía estar bien era:
```suggestion
  override fun daSemillas() = super.daSemillas() || altura > 0.4
```